roles/openshift_health_checker/callback_plugins/zz_failure_summary.py

```diff
@@ -95,7 +95,7 @@ def _print_check_failure_summary(self, failed_checks, context):
             'Variables can be set in the inventory or passed on the\n'
             'command line using the -e flag to ansible-playbook.\n'
         ).format(playbook=self._playbook_file, checks=checks)
-        if context in ['pre-install', 'health']:
+        if context in ['pre-install', 'pre-upgrade', 'health']:
```

Comment:
@sosiouxme when you designed this, do you remember what were the "contexts" in which the summary message below should not be printed?